docs/en/sql-reference/statements/insert-into.md

```diff
@@ -230,7 +236,15 @@ Performance will not decrease if:
 - Data is added in real time.
 - You upload data that is usually sorted by time.
 
-It's also possible to asynchronously insert data in small but frequent inserts. The data from such insertions is combined into batches and then safely inserted into a table. To enable the asynchronous mode, switch on the [async_insert](../../operations/settings/settings.md#async-insert) setting. Note that asynchronous insertions are supported only over HTTP protocol, and deduplication is not supported for them.
+### Asynchronous inserts
+
+It is possible to asynchronously insert data in small but frequent inserts. The data from such insertions is combined into batches and then safely inserted into a table. To use asynchronous inserts, enable the [`async_insert`](../../operations/settings/settings.md#async-insert) setting. Note that asynchronous inserts are supported only over HTTP protocol and that deduplication is not supported for them.
```

Comment:
They are supported in the native protocol as well, and deduplication is also supported.

Reply:
Ok just to clarify, deduplication is supported for both native protocol and HTTP protocol? So I should just remove the following sentence:

>Note that asynchronous inserts are supported only over HTTP protocol and that deduplication is not supported for them.

Reply:
Enabled explicitly now:
https://clickhouse.com/docs/en/operations/settings/settings#async-insert-deduplicate

Dedup window in inserts and time:
https://clickhouse.com/docs/en/operations/settings/merge-tree-settings#replicated-deduplication-window-for-async-inserts
https://clickhouse.com/docs/en/operations/settings/merge-tree-settings#replicated-deduplication-window-seconds-for-async_inserts